package/doc/sphinx/source/documentation_pages/analysis/hole2.rst

```diff
@@ -18,6 +18,18 @@ from http://www.holeprogram.org/ or as source from
 https://github.com/osmart/hole2. (HOLE is open source and available
 under the Apache v2.0 license.)
 
+
+.. deprecated:: 2.8.0
+  This module is deprecated in favour of the mdakit
+  `mdahole2 <https://www.mdanalysis.org/mdahole2/>`_ and
+  will be removed in MDAnalysis 3.0.0.
```

Comment:
```suggestion
   This module is deprecated in favour of the mdakit
   `mdahole2 <https://www.mdanalysis.org/mdahole2/>`_ and
   will be removed in MDAnalysis 3.0.0.
```
Is this indentation correct?

Reply:
I think it doesn't matter all that too much with sphinx, but I've added it, thanks!